Library/Homebrew/formulary.rb

```diff
@@ -133,6 +138,15 @@ def self.load_formula(name, path, contents, namespace, flags:, ignore_errors:)
       remove_const(namespace)
       raise new_exception, "", e.backtrace
     end
+  ensure
+    printed_to_stdout = $stdout.string.strip
+    if printed_to_stdout.present?
+      opoo <<~WARNING
+        Formula #{name} attempted to print the following while being loaded:
+        #{$stdout.string.strip}
+      WARNING
```

Comment:
If we can get a tap name, might be good to point users to file bug report to them.

Reply:
Yea, I wanted a tap name, but couldn't work out how to get it. Suggestions?

Reply:
Could maybe make it an error that bubbles up like: https://github.com/Homebrew/brew/blob/6141c6a181ed957f8d4291c8a1392328f741e8d5/Library/Homebrew/formulary.rb#L753-L754 (though ultimately caught and turned into `opoo` in the caller of `load_file`).